chapters/synchronous.tex

```diff
@@ -965,8 +965,8 @@ \subsection{Base-clock Partitioning}\doublelabel{base-clock-partitioning}
 \textbf{continuous-time partitions} according to the following
 properties:
 
-A variable u in \textbf{sample}(u) and a variable y in y =
-\textbf{hold}(ud) is in a continuous-time partition.
+A variable u in \lstinline!sample(u)!, a variable y in y =
+\lstinline!hold(ud)!, and a variable b in \lstinline!Clock(b, startInterval)! where b is of boolean type is in a continuous-time partition.
```

Comment:
Done for Boolean. For the variables that wasn't a new issue, and thus I don't think it should be changed just because it was close to a another change - and there is a trade-off: on one hand they are Modelica-code - but on the other hand it may look odd if such a large portion is inline-code.